docs/workflow/implementing/messaging/mobile-microsurveys.md

```diff
@@ -0,0 +1,182 @@
+
+- Desired responses: the number of desired survey responses, e.g. 1000. This is determined by the survey owner.
+
+```json
+{
+  "messages": {
+    "feature-print-microsurvey": {
+      "title": "microsurvey_prompt_printing_title"
+    }
+  }
+}
+
+```
+
+You can either provide a `string` containing text that you would like to show or a pre-landed string id, which is already localizable. The latter is the preferred way so the survey supports multiple languages.
+
+## Customize the sheet
+
+In the sheet we can control the question, answers and UTM parameters for the privacy policy link.
+
+For the question, just use the `text` field at the same level as the `title` one, similar to the latter where you can provide either a string or pre landed string id.
+
+```json
+{
+  "messages": {
+    "feature-print-microsurvey": {
+      "title": "microsurvey_prompt_printing_title",
+      "text": "microsurvey_survey_printing_title"
+    }
+  }
+}
+
+```
+
+To customize answers and UTM parameters, we have to add the `microsurvey-config` field object. For the answers just a new field called `answers`, it’s an array so each item will represent a possible answer, and it can contain two properties `text`, a string or pre-landed string of the text for the question and `ordering` which indicate in which position the possible answer should show.
+
+```json
+{
+  "messages": {
+    "feature-print-microsurvey": {
+      "title": "microsurvey_prompt_printing_title",
+      "microsurvey-config": {
+        "utm-content": "homepage",
+        "answers": [
+          {
+            "text": "likert_scale_option_1",
+            "ordering": 0
+          },
+          {
+            "text": "likert_scale_option_2",
+            "ordering": 1
+          },
+          {
+            "text": "likert_scale_option_3",
+            "ordering": 2
+          },
+          {
+            "text": "likert_scale_option_4",
+            "ordering": 3
+          },
+          {
+            "text": "likert_scale_option_5",
+            "ordering": 4
+          },
+          {
+            "text": "likert_scale_option_6",
+            "ordering": 5
+          }
+        ]
+      }
+    }
+  }
+}
+
+```
+
+The icon can be added by using the field `icon` under `microsurvey-config`, you have to provide a valid icon resource id, a full list of possible icons can be found [here](https://searchfox.org/mozilla-central/source/mobile/android/android-components/components/ui/icons/src/main/res/drawable) and [here](https://searchfox.org/mozilla-central/source/mobile/android/fenix/app/src/main/res/drawable).
+Something to keep in, mind is Android usages a special type vector called [vector drawables](https://developer.android.com/develop/ui/views/graphics/vector-drawable-resources), to visualize it you will need Android studio with Fenix source code, ideally if the icon hasn't been use before confirm with the Android team how it will look like.
```

Comment:
```suggestion
Something to keep in, mind is Android usages a special type vector called [vector drawables](https://developer.android.com/develop/ui/views/graphics/vector-drawable-resources), to visualize it you will need Android studio with Fenix source code, ideally if the icon hasn't been use before confirm with the Android team how it will look.
```